config.json

```diff
@@ -560,6 +560,18 @@
         "strings"
       ]
     },
+    {
+      "slug": "palindrome-products",
+      "uuid": "bc74b87d-d257-41e5-9233-10324a78367c",
+      "core": false,
+      "unlocked_by": "accumulate",
+      "difficulty": 6,
+      "topics": [
+        "algorithms",
+        "mathematics",
```

Comment:
Per the recent discussion in https://github.com/exercism/exercism.io/issues/4110, would you mind making this 'math' instead of 'mathematics'?

Reply:
Ok, np, I made the requested changes :)